invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/Home/components/widgets/DocumentsCard/DocumentsCard.js

```diff
@@ -7,18 +7,22 @@ import { BackOfficeRoutes } from '../../../../../../routes/urls';
 import { document as documentApi } from '../../../../../../common/api';
 import { goToHandler } from '../../../../../../history';
 
-export default class DocumentsCard extends Component {
+class DataCard extends Component {
```

Comment:
are you planning to reuse it?

Reply:
No that's why is not exported. It's just to handle properly the rendering of `DocumentsCard`. I can rename it in `DocumentsDataCard` if you prefer.